### PR TITLE
Added Via keymap to Spaceman 2% Milk

### PR DESCRIPTION
**Pull request by wongyuhao**
## Description

Added Via keymap to support Spaceman 2% Milk 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keymap/layout/userspace (addition or update)
- [ ] Keyboard (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/spaceman/2_milk/config.h [hunk 1]
```diff
@@ -17,8 +17,8 @@
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0xB195
+#define VENDOR_ID       0x5350
+#define PRODUCT_ID      0x3225
```
Comment (fauxpark) [inline keyboards/spaceman/2_milk/config.h:21]:
```suggestion
#define VENDOR_ID       0x5350 // "SP"
#define PRODUCT_ID      0x3225 // "2%"
```

File: keyboards/spaceman/2_milk/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+      [2] = LAYOUT(
+      KC_TRNS,
+      KC_TRNS
+      ),
+
+      [3] = LAYOUT(
+      KC_TRNS,
+      KC_TRNS
+      )
```
Comment (fauxpark) [inline keyboards/spaceman/2_milk/keymaps/via/keymap.c:22]:
```suggestion
      [0] = LAYOUT(
          KC_Z,
          KC_X
      ),
      [1] = LAYOUT(
          _______,
          _______
      ),
      [2] = LAYOUT(
          _______,
          _______
      ),
      [3] = LAYOUT(
          _______,
          _______
      )
```

File: keyboards/spaceman/2_milk/config.h (hunk 1 above)
Comment (Spaceman) [reply keyboards/spaceman/2_milk/config.h:21]: id prefer SB here

**Comment by keyboard-magpie** [comment]
nudging @wongyuhao

File: keyboards/spaceman/2_milk/config.h (hunk 1 above)
Comment (noroadsleft) [reply keyboards/spaceman/2_milk/config.h:21]: ... which would make this:

```suggestion
#define VENDOR_ID       0x5342 // "SB"
#define PRODUCT_ID      0x3225 // "2%"
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**